src/BotService/custom/New-AzBotService.ps1

```diff
@@ -81,7 +81,7 @@ function New-AzBotService {
         [Parameter(Mandatory)]
         [Microsoft.Azure.PowerShell.Cmdlets.BotService.Category('Path')]
         [System.String]
-        ${Location},
+        ${Location} = 'global',
```

Comment:
If Location is mandatory parameter, why shall we set a default value?

Reply:
Made the parameter optional.